core/app/models/spree/order_contents.rb

```diff
@@ -40,6 +40,7 @@ def after_add_or_remove(line_item, options = {})
       shipment.present? ? shipment.update_amounts : order.ensure_updated_shipments
       PromotionHandler::Cart.new(order, line_item).activate
       Adjustable::AdjustmentsUpdater.update(line_item)
+      TaxRate.adjust(order, [line_item]) if line_item.quantity == 1
```

Comment:
It is not so clean, but it will do the trick, at least as temporarily fix. I will update pull request.